docs/0002-fixed-length-n-tuple-trees.md

```diff
@@ -0,0 +1,220 @@
+# OCFL Community Extension 0002: N-tuple Trees for OCFL Storage Hierarchies
```

Comment:
I think this should be "Fixed Length N-tuple Trees for OCFL Storage Hierarchies"

Reply:
Yep!